### PR TITLE
Publishing

### PR DESCRIPTION
**Pull request by fommil**
the infinite digression of just wanting to make a code change but realising that the CI is not setup to publish yet... *sigh*

### CODE REVIEW

**Comment by fommil** [comment]
@Tomahna FYI, if you're around a review would be :100:

**Comment by fommil** [comment]
oh wow, I found the button for gitlab! We now have options.

**Comment by fommil** [comment]
nope, gitlab didn't show up after all. I'm not sure how to get it working for PRs so travis seems the best choice.

**Comment by fommil** [comment]
I think I made a mistake with the secret envavrs.

File: .travis.yml [hunk 1]
```diff
@@ -2,78 +2,69 @@ os: linux
...
-      brew install sbt;
-      export JAVA_HOME=$(/usr/libexec/java_home);
-    fi
+- if [[ "$TRAVIS_OS_NAME" = "osx" ]]; then brew update; brew install sbt; export JAVA_HOME=$(/usr/libexec/java_home);
+  fi
 stages:
-  - name: test
-  - name: integration
-  - name: publish
-    if: branch = master
```
Comment (Tomahna) [inline .travis.yml]: actually this is probably a mistake I introduced as ensime does not have a master branch

Comment (Tomahna) [reply .travis.yml]: (should be 3.0)

Comment (fommil) [reply .travis.yml]: ah, I can fix that here, well spotted

**Comment by Tomahna** [comment]
have you tried scoping them to the publish task ?
it's the only one which need them right ?

**Comment by Tomahna** [comment]
But for secret variables, I'd go with the web ui though.

**Comment by fommil** [comment]
hmm, the publish task is running in the PR, not on merge, so I think there is a typo here. I'll disable the publish task until we get that fixed.

**Comment by Tomahna** [comment]
It's weird cause https://github.com/Tomahna/ensime-server/pull/1 didn't trigger publish.

**Comment by Tomahna** [comment]
nevermind

**Comment by fommil** [comment]
well this is already a big improvement, I'll change the ticket to be "get it publishing" which should be a smaller scope!

**Pull request merged**